### PR TITLE
Add airflow-provider-xlsx 1.0.1

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@xylar this is the only provider for xlsx I could find and was listed on https://registry.astronomer.io/providers/xlsx
I have added you as a maintainer, can you confirm being a maintainer?

File: recipes/airflow-provider-xlsx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+about:
+  home: https://github.com/andreax79/airflow-provider-xlsx
+  summary: Airflow operators for reading and writing XLSX files
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - xylar
```
Comment (xylar) [inline recipes/airflow-provider-xlsx/meta.yaml:44]: I agree to be a maintainer.

**Comment by rxm7706** [comment]
@conda-forge/staged-recipes  ready for review

**Comment by rxm7706** [comment]
@conda-forge/help-python  this is ready to review. The Windows failures are expected because apache-airflow doesn't support Windows.

**Comment by rxm7706** [comment]
@conda-forge/help-python this is ready to review. The Windows failures are expected because apache-airflow doesn't support Windows.

**Review by dopplershift — APPROVED** [review]
Ignoring noarch Windows build failure due to missing dependency.

**Comment by xylar** [comment]
Thanks @dopplershift!

**Pull request merged**